src/cascadia/TerminalApp/PaletteItem.cpp

```diff
@@ -15,4 +15,11 @@ using namespace winrt::Windows::System;
 
 namespace winrt::TerminalApp::implementation
 {
+    Controls::IconElement PaletteItem::ResolvedIcon()
+    {
+        const auto icon = IconPathConverter::IconWUX(Icon());
+        icon.Width(16);
+        icon.Height(16);
+        return icon;

```

Comment:
```suggestion
        return IconPathConverter::IconWUX(Icon());
```
Do you even need to resize it here? It seems like you already have it set to 16x16 all throughout CommandPalette.xaml. Might as well have `ResolvedIcon()` return the icon itself, and the view (XAML) be responsible for resizing it to the appropriate size.

Reply:
So the `ContentPresenter` in XAML is set to 16x16, but the `IconElement` that ends up being the content _inside_ the ContentPresenter also needs to also be 16x16 otherwise it looks weird